.gitignore

```diff
@@ -1,3 +1,5 @@
+.DS_Store
+.idea/

```

Comment:
IDE specific project files should be excluded in your global `.gitignore`, not in project repositories. I'd say the same applies to `nbproject`.